mvrx/src/main/kotlin/com/airbnb/mvrx/CoroutinesStateStore.kt

```diff
@@ -76,12 +77,12 @@ class CoroutinesStateStore<S : MvRxState>(
      * }
      */
     @Synchronized
```

Comment:
`@Synchronized` on suspend fun will not work properly, plus since it's necessary only for sync case, probably would be better to move it to only blocking call